arithmetization/src/main/java/net/consensys/linea/zktracer/module/hub/fragment/imc/ImcFragment.java

```diff
@@ -137,7 +137,14 @@ public Trace trace(Trace trace) {
     trace.peekAtMiscellaneous(true);
 
     for (TraceSubFragment subFragment : moduleCalls) {
-      subFragment.trace(trace, hub.state.stamps());
+      if (subFragment instanceof MmuCall) {
+        MmuCall mmuCall = (MmuCall) subFragment;
+        if (mmuCall.traceMe()) {
+          subFragment.trace(trace, hub.state.stamps());
+        }
+      } else {
+        subFragment.trace(trace, hub.state.stamps());
+      }
```

Comment:
This fixes an issue where the HUB would trace the MMU instruction for reverted LOG's